roles/confluent.variables/vars/main.yml

```
@@ -258,7 +271,7 @@ schema_registry_properties:
       ssl.keystore.password: "{{schema_registry_keystore_storepass}}"
       ssl.key.password: "{{schema_registry_keystore_keypass}}"
   truststore:
-    enabled: "{{schema_registry_ssl_mutual_auth_enabled|bool or mds_ssl_enabled|bool}}"
+    enabled: "{{schema_registry_ssl_mutual_auth_enabled|bool or mds_ssl_enabled|bool or mds_broker_listener.ssl_mutual_auth_enabled|bool}}"
```

```suggestion
    enabled: "{{schema_registry_ssl_mutual_auth_enabled|bool or mds_ssl_enabled|bool}}"
```

sr should not be impacted by the mds listener and there are sr properties below for when mds ssl is enabled